docs/forecast.md

```diff
@@ -171,6 +171,10 @@ pageClass: routes
 
 </Route>
 
+### 深圳市
+<Route author="lilPiper" example="/tingshuitz/shenzhen" path="/tingshuitz/shenzhen">
+可能仅限中国大陆服务器访问，以实际情况为准。
```

Comment:
Each line must be spaced.
Missing closing tag.
docs: https://docs.rsshub.app/joinus/new-rss/add-docs.html